config/service_providers.yml

```diff
@@ -363,3 +363,16 @@ production:
       - dob
       - ssn
       - phone
+
+  # CBP GOES
+  'urn:gov:dhs.cbp.jobs:openidconnect:jenkins-pspd-credential-service':
+    friendly_name: 'CBP PSPD Trusted Traveler Programs'
+    agency: 'DHS'
+    logo: 'cbp.png'
+    cert: 'cbp_goes_pre_prod'
+    redirect_uris:
+      - 'http://10.156.152.27/login'
+    attribute_bundle:
+      - email
+      - first_name
```

Comment:
1. attribute bundle doesn't matter for OIDC yet
2. they are LOA1 so only `email` is accessible